### PR TITLE
#40 added global error handler on APIException

### PR DESCRIPTION
**Pull request by nickonos**

### CODE REVIEW

**Review by NealGeilen — APPROVED** [review]

**Review by DamienKusters — APPROVED** [review]

**Comment by DamienKusters** [comment]
There seem to be some merge conflicts, please resolve these prior to merging.

File: lpr.Logic/Services/OrganisationService.cs [hunk 1]
```diff
@@ -44,5 +45,15 @@ public class OrganisationService : IOrganisationService {
...
       return org;
     }
+
+    public async Task<List<Organisation>>
+    GetOrganisationsPaginatedAsync(int amount, Guid? lastOrganisationId) {
+      return await _organisationData.GetOrganisationsPaginatedAsync(
+          amount, lastOrganisationId);
+    }
+
+    public Organisation GetOrganisation(string OrgId) {
```
Comment (DamienKusters) [inline lpr.Logic/Services/OrganisationService.cs]: There seem to be two methods called 'GetOrganisation' in this file.

Comment (DamienKusters) [reply lpr.Logic/Services/OrganisationService.cs]: This one can be deleted.

**Review by DamienKusters — CHANGES_REQUESTED** [review]

File: lpr.Logic/Services/OrganisationService.cs [hunk 2]
```diff
@@ -18,22 +18,37 @@ public class OrganisationService : IOrganisationService {
       _organisationData = new OrganisationData(ctx);
     }
 
-    public int AddOrganisation(string Name, string UserId) {
+    public void AddOrganisation(string Name, string UserId) {
```
Comment (DamienKusters) [inline lpr.Logic/Services/OrganisationService.cs:21]: Maybe return something here, to confirm that AddOrganisation actually got processed.

**Review by DamienKusters — APPROVED** [review]

**Review by WouterPennings — APPROVED** [review]

**Pull request merged**